docker-compose.override.yml

```diff
@@ -101,6 +101,7 @@ services:
       - "9200:9200"
     environment:
       - ELASTIC_PASSWORD=test
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
https://github.com/ModularHistory/modularhistory/blob/6ff333233af734f9212f6c1f6175801d78750f45/docker-compose.yml#L162

Reply:
@alashow it limits the heap size, which does not include statically allocated memory. For me, it reduced memory consumption from ~2G to ~1G.